zombies.js

```diff
@@ -8,6 +9,21 @@
  * @property {string} name
  */
 
+class Item {
+  constructor(name) {
+    typeErrors(name, 'string');
+    this.name = name;
+  }
+}
+
+function typeErrors(objectToCheck, typeString) {
+  if ((typeof objectToCheck) !== typeString) {
```

Comment:
When you end up checking a value multiple time, that is a good place to use a variable.
```js
  const inputType = typeof objectToCheck;
  if (inputType !== typeString)
```